README.md

```diff
@@ -1,2 +1,82 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Portfolio
-My portfolio that shows my educational background, projects and technical skills.
+
+> This is a mobile-first portfolio project which is built to show projects, background and the ways to get in touch with the developer.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+N/A
+
+## Get Started
+
+To get the content of this project locally you need to run the following commands in your terminal:
+
+### Prerequisites
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/Khawanin/Portfolio
+```
+3. To access cloned directory run:
+```bash
+cd Portfolio
+```
+3. Open it with the live server
+
+### Install
+> To install linters and other project's dependencies run:
+```bash
+npm install
+```
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Authors
+
+- GitHub: [@khawanin](https://github.com/Khawanin)
+- LinkedIn: [@khawaninali](https://www.linkedin.com/in/khawanin-ali-zada-93777b21a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Khawanin/Portfolio/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thank You [Microverse](www.microverse.org) team that help me with the template of this project.
+- Thanks to everyone who will get time to check this code and suggest any [issue](https://github.com/Khawanin/Portfolio/issues) to improve the page.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [Optional] Please consider adding [this MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md) License file in this repository. :)